### PR TITLE
Add group 8's literature review.

### PR DESCRIPTION
**Pull request by olepbr**
Hi Luis,

Please find enclosed my group's literature review. I tested the rendering by building and running the Docker container specified in the readme, so should be good to go; let me know if you want anything amended!

Side note: I believe building & running the container should be the suggested way of testing, as opposed to asking folks to install Jekyll manually. Just installing Jekyll isn't enough anyway, as you then have to do `bundle install`, which fails (at least for me) as it tries to install the gems into `/usr/lib`, which is reasonable as you probably don't want a bunch of random gems in there. =)
In other words, I think [literature_review.md](https://github.com/luiscruz/course_sustainableSE/blob/bd9e1625f2ef0cd4149aa5c830bdfbee9f7e7df5/2023/literature_review.md) should be updated.

### CODE REVIEW

**Comment by luiscruz** [comment]
Hi @olepbr thanks for the submission! Indeed the docker container had an issue with the ruby and the jekyll versions. I thought I fixed it yesterday, but let me know if that was not the case. (I should have done it earlier, I was overly optimistic about it)

Edit: Reading again your comment, you're referring to the guidelines. Indeed, it still mentions the non-docker approach. My bad!. Thanks for bringing it up!!

**Comment by olepbr** [comment]
> (I should have done it earlier, I was overly optimistic about it)

Just like I was overly optimistic about making it to class on time :–)

> Edit: Reading again your comment, you're referring to the guidelines. Indeed, it still mentions the non-docker approach. My bad!. Thanks for bringing it up!!

Yes, indeed, I was just referring to the guidelines, not the Docker setup. Saw that you fixed it in 23189bc, nice!
